cryptoinvestor/models/user.py

```diff
@@ -1,4 +1,4 @@
-from cryptoinvestor.models.Account import Account
+from cryptoinvestor.models.account import Account
```

Comment:
Why the lowercases ? Is it a general convention ?

Reply:
It is a general convention according to [PEP 8](https://www.python.org/dev/peps/pep-0008/#package-and-module-names)

> Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability. Python packages should also have short, all-lowercase names, although the use of underscores is discouraged.